apps/web/src/hooks/useGetWishlist.ts

```
@@ -1,10 +1,16 @@
+import { useSuspenseInfiniteQuery } from '@tanstack/react-query';
 
 import { getWishlist } from '@/apis/getWishlist';
 
 export const useGetWishlist = () => {
-  return useSuspenseQuery({
+  const { data, fetchNextPage, hasNextPage, isFetchingNextPage } = useSuspenseInfiniteQuery({
     queryKey: ['wishlists'],
-    queryFn: getWishlist,
+    queryFn: ({ pageParam }) => getWishlist(pageParam),
+    initialPageParam: null as string | null,
+    getNextPageParam: page => (page.hasNext ? page.nextCursor : undefined),
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)